.gitignore

```diff
@@ -1,4 +1,4 @@
 **/.package-lock.json
 /.nyc-output
+/.nyc_output
 /node_modules
-/package-lock.json
```

Comment:
This is a primarily a CLI app. And adding a lockfile makes the CI builds reproducible - I don't think it's beneficial to have every run of CI use potentially different set of dependencies.

Reply:
Making builds reproducible for CI, but not for consumers, is actively hostile - and including a shrinkwrap file is also hostile, since it prevents deduping.

Reply:
npm ignores package-lock.json on publish, and this project doesn't have a shrinkwrap file, so I think this won't change anything for the library consumers?

Reply:
Right, that's the point. Library consumers get an unlocked experience, so it's very wrong for project developers to get a different experience.